src/bootstrap/src/core/build_steps/gcc.rs

```diff
@@ -41,9 +42,6 @@ pub fn prebuilt_gcc_config(builder: &Builder<'_>, target: TargetSelection) -> Gc
     // Initialize the gcc submodule if not initialized already.
     builder.config.update_submodule("src/gcc");
 
-    // FIXME (GuillaumeGomez): To be done once gccjit has been built in the CI.
-    // builder.config.maybe_download_ci_gcc();
```

Comment:
I would think that we still want to add support for this at some point.

Reply:
Sure! That's one of the follow-up steps, but it will probably look a bit different than this (I hope, because stuffing even more things into Config isn't great), so I just removed this commented code for now.